drivers/sensor/lis2dw12/lis2dw12.c

```diff
@@ -498,13 +550,15 @@ static int lis2dw12_init(const struct device *dev)
 					   0),				\
 		},							\
 		.pm = DT_INST_PROP(inst, power_mode),			\
+		.odr = DT_INST_PROP(inst, odr),			\
```

Comment:
Use DT_INST_PROP_OR() and remove `default:` from the binding.

Reply:
resolved.